ArcGISRuntimeSDKQt_QMLSamples/CloudAndPortal/IntegratedWindowsAuthentication/IntegratedWindowsAuthentication.qml

```diff
@@ -0,0 +1,220 @@
+// [WriteFile Name=IntegratedWindowsAuthentication, Category=CloudAndPortal]
+// [Legal]
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+import Esri.ArcGISRuntime 100.6
+import Esri.ArcGISRuntime.Toolkit.Dialogs 100.6
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    readonly property url arcgis_url: "https://www.arcgis.com"
+    property var portalItem
+    property var iwaSecurePortal
+
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        Map {
+            BasemapTopographic {}
+        }
+    }
+
+    Portal {
```

Comment:
this looks much better. I think the IWA secured Portal instance should probably be declared as well. Is this not done because you can't re-set the URL in the case that the user provides incorrect URL?

Reply:
Precisely.